.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": false,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "arrowParens": "avoid"
+}
```

Comment:
I assume you're not changing any defaults with this config, in which case, why bother? I'm happy with Prettier's defaults.

Reply:
Correct, this is the default prettier config. However, I think I’ve modified my VS Code prettier settings to something different, so when I save (I format on save), it uses those settings instead. Hence this file.